synapse/tests/files/stormpkg/testpkg.yaml

```diff
@@ -0,0 +1,28 @@
+name: testpkg
+version: 0.0.1
+
+modules:
+    - name: testmod
+
+external_modules:
+    - name: testext
+      package: synapse.tests.files
+      package_path: stormmod/common
+
+    - name: testextfile
+      file_path: synapse/tests/files/stormmod/common
```

Comment:
Since we're in a yaml file we could add a comment noting that this is the relative path from where the genpkg tool is going to be executed from. Reminder for future selves.

Reply:
Added a comment